src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Colors;
+import core.basesyntax.Figure;
+import core.basesyntax.FigureArea;
+
+public class Circle extends Figure implements FigureArea {
```

Comment:
```suggestion
public class Circle extends Figure {
```
Figure already implements interface